### ISSUE TITLE
[Validator] (#2) Generate Problematic Images

### ISSUE THREAD

**Issue by piercefreeman** [cs210/solvence-code#9]
## What problem are we trying to solve?

Users frequently take images that can't be validated because of deteriorated image quality. This could be because they're too small, blurry, too dark, too bright, etc.  To create a training set for this purpose, we can randomly generate permuted images from a known base image.  Strategies include -
- Applying noise
- Changing brightness
- Adding blur
- Rotation / Perspective Transformation

We should also collect a test set from real images with known issues.  This may require manually tagging a small set of photos from online resources.

## How do we measure success

- Generate a training set of ~30,000 examples for our deep learning model with an automated script
- Find a storage mechanism that's well suited to hold our examples (ideally on Oracle that can be spun up via terraform)

**Comment by piercefreeman** [comment]
@samjeckert Could you write up a small wiki page on how our terraform infrastructure is set up?

**Comment by piercefreeman** [comment]
Implemented by @geoffreyangus in #22
